src/MPS.jl

```diff
@@ -126,14 +126,26 @@ function MPS(::Canonical, arrays, λ; order=defaultorder(MPS), check=true)
     return mps
 end
 
+"""
+    check_form(mps::AbstractMPO)
+
+Check if the tensors in the mps are in the proper [`Form`](@ref).
+"""
 check_form(mps::AbstractMPO) = check_form(form(mps), mps)
 
 function check_form(config::MixedCanonical, mps::AbstractMPO)
     orthog_center = config.orthog_center
+
+    left, right = if orthog_center isa Site
+        id(orthog_center) .+ (0, 0)
```

Comment:
mmm remove the add?
```suggestion
        id(orthog_center)
```

Reply:
This is needed to unpack the same value to both `left` and `right` 🥶

Reply:
ah okay, can you put a comment then?

Reply:
oki